DM_Applications_Design.tex

```diff
@@ -139,7 +139,7 @@
 
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 %%% Command's for merging Robert's document, at least temporarily.
-%\usepackage[top=1cm, bottom=2cm, left=2cm, right=2cm]{geometry} 
```

Comment:
It's not super important, but we usually try to keep whitespace changes in a separate commit unrelated to other changes.  This is more important in code, but I thought I'd mention it.